operations/bi.go

```diff
@@ -0,0 +1,17 @@
+package operations
+
+import "github.com/beyondstorage/go-storage/v4/types"
+
+type BiOperator struct {
+	src   types.Storager
+	dst   types.Storager
+	errCh chan error
+}
+
```

Comment:
Do we need to consider in advance how to handle batch operations (variable parameters)?

Reply:
We didn't consider this for now. But it may be an enhancement in the future if someone really needed.

Reply:
Got it.